Add Steps to Setup Jaeger-UI on WSL (#1612)
jaegertracing/jaeger-ui: README.md

See [CONTRIBUTING](./CONTRIBUTING.md).

> Stuck somewhere or found a bug? [Create a GitHub Issue](https://github.com/jaegertracing/jaeger-ui/issues/new/choose), or try asking for help in the [Slack Channel](https://www.jaegertracing.io/get-in-touch/)

Reviewer: ```suggestion
Stuck somewhere or found a bug? See [Getting in Touch](https://www.jaegertracing.io/get-in-touch/) on how to ask for help.
```

ssh -fN -L 16686:$BACKEND_HOST:$BACKEND_PORT $BACKEND_HOST
```

> If you are using [UI Base Path](https://www.jaegertracing.io/docs/1.7/deployment/#ui-base-path) feature, you need to append the base path into `jaeger-ui/jaeger-ui/vite.config.js` in ```proxyConfig``` object. for example: if the base path is `"/jaeger"`, then the ```target``` should be `"http://localhost:16686/jaeger"`

Reviewer: GitHub markdown uses a single back-tick for inline code-formatted text. Three back-ticks is for code blocks.

Perhaps it would make sense to show an example snippet from the config.

The above command will run a web server on port :5173 that will serve the UI assets, with hot reloading support, and it will proxy all API requests to `http://localhost:16686` where Jaeger query should be running.

> ------------

Reviewer: ```suggestion
```

> ### Using Windows OS for Development?

Reviewer: it would more sense to have this as the 2nd subsection under Development. And the 1st subsection should be Prerequisites that lists the requirements, namely `nvm`, Node.js + `npm`, and `yarn` (don't think we need anything else).

While we don't natively support Windows OS for running the Jaeger UI Dev Environment, you can use WSL to run it.

Here are some steps to follow:
1. Install WSL: https://learn.microsoft.com/en-us/windows/wsl/install

> 2. Install Node.JS: https://learn.microsoft.com/en-us/windows/dev-environment/javascript/nodejs-on-wsl

Reviewer: What about `yarn`? Any special instructions for it?

Author: It used to come pre-installed in Ubuntu 22 over WSL but I don't remember it exactly, so I have included a guide for it too.

3. Connect WSL Environment with VSCode: https://learn.microsoft.com/en-us/windows/dev-environment/javascript/nodejs-on-wsl#install-visual-studio-code
4. Use the WSL Terminal inside VSCode and [follow the Jaeger UI installation steps](https://github.com/jaegertracing/jaeger-ui#running-the-application)

> Stuck somewhere or found a bug? [Create a GitHub Issue](https://github.com/jaegertracing/jaeger-ui/issues/new/choose), or try asking for help in the [Slack Channel](https://cloud-native.slack.com/archives/CGG7NFUJ3)

Reviewer: this isn't specific to Windows section. I would add it to the Contributing section above.

Also, instead of a direct link to Slack, we prefer https://www.jaegertracing.io/get-in-touch/

> ---------------

Reviewer: ```suggestion
```
